src/routes/recommendations.py

```diff
@@ -67,14 +67,26 @@ def recommendations(
     findings = finding_repository.get_findings_by_task_id(task.id, request.pagination)
 
     total_count = finding_repository.get_findings_count_by_task_id(task.id)
+    
+    recommendations = [
+    db_finding_to_response_item(find)
+    for find in findings
+    if severity_range is None or (
+        (response_item := db_finding_to_response_item(find)).severity is not None and
+        len(severity_range) == 2 and
+        severity_range[0] <= response_item.severity <= severity_range[1]
```

Comment:
It probably makes sense to do it on the database level than the application level. We have severity stored in findings.